### PR TITLE
Update icons size and fixed over scroll

### PR DESCRIPTION
**Pull request by tshlos**
Hey Diego, just did some minor changes on icon properties and wrote comments on the lines I changed.

### CODE REVIEW

**Review by Deohgu — CHANGES_REQUESTED** [review]
### Game.jsx :
The margin right was working as intended.
_marginRight_ is the camel cased version of margin-right.
You can read about it [here](https://www.pluralsight.com/guides/react-inline-styling).

I think that issue might just be connected with the over-scrolling. So maybe fixing that first with solve this one.

---

### Game.styled.js :
The over scrolling issue seems to remain, the width or height don't go over 100vw/vh so that shouldn't be a problem. The 80vw amount is mostly a stylistic a choice. 
So something else must be causing extra space that pushes over the page size.
Maybe try removing the margins of GameStyled, and then it can be re-center using Flex on the parent component instead.

**Comment by tshlos** [comment]
Hi Diogo, do you mind sharing a screenshot of what you see so I can have a
better idea? Thanks

**Comment by Deohgu** [comment]
Of course.
Sorry, maybe over scroll wasn't the best term. So when the window width decreases to 800px, the height of the app becomes larger than the window and overflows so you get the vertical scroll bar. I'm assuming that it might be due to the extra margins.
The intend behaviour is the app would stay centred without overflowing.

**Comment by tshlos** [comment]
I see.. is it ok if I still work on this? Thanks for sharing the screenshot.

**Comment by Deohgu** [comment]
Yes that's perfectly fine.
Have a look at my initial review of your Pull-Request for ideas.
When you're finished just request another review.

**Comment by tshlos** [comment]
Will do, thanks!

**Review by Deohgu — CHANGES_REQUESTED** [review]
I've commented on the specific parts of the code.

File: src/components/Game/Game.styled.js [hunk 1]
```diff
@@ -1,33 +1,35 @@
...
-  margin: 20% auto;
+  position: fixed;
+  display: block;
+  margin: auto;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  top: 0;
+  width: 100%;
+  height: 100%;
```
[the remark this answers is not in the document]
Comment (tshlos) [reply src/components/Game/Game.styled.js:12]: I see what you mean. Thanks for the suggestion.

**Comment by tshlos** [comment]
Hey Diego, I just updated with the changes you suggested. Let me know what
you think.

**Review by Deohgu — APPROVED** [review]
Looks great, good job!
Ready to merge with the master.

**Comment by tshlos** [comment]
Awesome!

**Pull request merged**